### ISSUE TITLE
Level 2 crashes

### ISSUE THREAD

**Issue by jmtstorres** [ja-brouil/TBS_GoDot#12]
Game runs as expected at the beginning, but when selecting new game and watching the first cutscene the game crashes with the message:
_**E 0:00:20.544   _load: Failed loading resource: res://assets/levels/level3/level3.tmx. Make sure resources have been imported by opening the project in the editor at least once.
  <Erro C++>    Condition "found" is true. Returned: RES()
  <Origem C++>  core/io/resource_loader.cpp:278 @ _load()
  <Rastreamento de pilha>Chapter Background.gd:31 @ start()**_
Latest version recently downloaded, to reproduce just download the zip file and run. I am trying using Godot 3.2.3 and Running in Linux MInt 20.

**Comment by ja-brouil** [comment]
Is there a a level3.tmx file in that folder?

**Comment by pawngames** [comment]
Yes, it is a fresh download. No changes were made to the project structure whatsoever.

**Comment by ja-brouil** [comment]
I've tested both using a zip and git clone using terminal and both times the game level loads fine for me. Can you navigate to that level3 folder and see is the file is in there and if the permissions are set correctly? I tested on windows and mac.

**Comment by pawngames** [comment]
![image](https://user-images.githubusercontent.com/76188858/103180133-ab74e080-4871-11eb-8333-0f5997037145.png)
From the file manager seems ok, but Godot apparently has an issue with this file.
![image](https://user-images.githubusercontent.com/76188858/103180149-d52e0780-4871-11eb-9063-523921533b86.png)
These are the import properties, i really can't see anything wrong, but trying to edit it in Godot also gives me an error. The other levels are opening fine.
![image](https://user-images.githubusercontent.com/76188858/103180189-250cce80-4872-11eb-9d97-6849b71e0aa6.png)

**Comment by pawngames** [comment]
I tried again with a fresh download and tinkered with the tmx file that states error and the only thing i saw that caught my eye was some japanese chars on some properties, but i don't think that could be it. The xml file is ok, Godot just doesn't want to open or make it valid.

**Comment by ja-brouil** [comment]
Are you able to open the tmx file in an editor like VS code?

I’m just unable to reproduce this error but it could be because of the Japanese characters in that level
Sent from my iPhone

On Dec 27, 2020, at 2:35 PM, pawngames <notifications@github.com> wrote:

﻿

I tried again with a fresh download and tinkered with the tmx file that states error and the only thing i saw that caught my eye was some japanese chars on some properties, but i don't think that could be it. The xml file is ok, Godot just doesn't want to open or make it valid.

**Comment by pawngames** [comment]
Removed the chars but still get the error. Maybe Godot is caching something wrong for that matter. 
The chars are in the main branch, but since MacOS is linux-like i would have thought it should occur too:
![image](https://user-images.githubusercontent.com/76188858/103182200-6c9e5500-4888-11eb-8177-1f666544e6e6.png)
Will try to fix the file before opening the project in Godot.

**Comment by ja-brouil** [comment]
Could you try downloading the project again, open it, delete the existing tmx file and replacing it with one from the newly downloaded project? This will force GoDot to reimport it


Sent from my iPhone

On Dec 27, 2020, at 4:16 PM, pawngames <notifications@github.com> wrote:

﻿

Removed the chars but still get the error. Maybe Godot is caching something wrong for that matter.
The chars are in the main branch, but since MacOS is linux-like i would have thought it should occur too:
[image]<https://user-images.githubusercontent.com/76188858/103182200-6c9e5500-4888-11eb-8177-1f666544e6e6.png>
Will try to fix the file before opening the project in Godot.
